services/core-web/src/components/Forms/ExplosivesPermit/ExplosivesPermitForm.tsx

```diff
@@ -117,13 +119,119 @@ export const ExplosivesPermitForm: FC<ExplosivesPermitFormProps &
     "now_application_guid"
   );
 
-  const isHistoric = !initialValues?.explosives_permit_id && props.isPermitTab;
+  const [isHistoric, setIsHistoric] = useState<boolean>(!initialValues?.explosives_permit_id && props.isPermitTab);
   const isESUP = props.userRoles.includes(USER_ROLES[Permission.EDIT_EXPLOSIVES_PERMITS]);
   // eslint-disable-next-line no-unused-vars
   const hasEditPermission = isESUP;
   // TODO: See MDS-5201- editing currently disabled
   const disabled = isProcessed; // isProcessed && !hasEditPermission;
+
+  const [radioSelection, setRadioSelection] = useState<number>(props.isPermitTab ? 1 : 2);
+  const [parentView, setParentView] = useState<boolean>(true);
+  const [isAmend, setIsAmend] = useState<boolean>(false);
+  const { isFeatureEnabled } = useFeatureFlag();
+
+  const handleRadioChange = (e) => {
+    setRadioSelection(e.target.value);
+    setIsHistoric(e.target.value == 1);
+    setIsAmend(e.target.value==3);
+  };
+
+  const handleOpenAddExplosivesPermitModal = () => {
+    setParentView(false)
+  }
+
+  const descriptionListElement = (
+    <div>
+      <Typography.Paragraph>
+        <ul className="landing-list">
+          <li>
+            <Typography.Text strong>Add an existing permit </Typography.Text>
+            <Typography.Text>
+              that was previously issued but does not exist in CORE and Minespace. This will help you keep track of your
+              past permits and activities.
+            </Typography.Text>
+          </li>
+          <li>
+            <Typography.Text strong>Create a new permit </Typography.Text>
+            <Typography.Text>this is meant for new explosive storage and use permits.</Typography.Text>
+          </li>
+          <li>
+            <Typography.Text strong>Amend an existing permit </Typography.Text>
+            <Typography.Text>
+              that has already been added to CORE and Minespace. This will allow you to make changes to your permit
+              conditions, such as the dates, amount of explosives.</Typography.Text>
+          </li>
+        </ul>
+      </Typography.Paragraph>
+    </div>
+  );
+
+  const amendDescriptionListElement = (
+    <div>
+      To make changes to an existing explosive storage and use permit, follow these steps:
+      <br />
+      <ul className="landing-list">
+        <li>Open the permit that you want to amend from the applications page of the mine in CORE.</li>
+        <li>Click on the “Amend Permit” button at the top right corner of the permit details page.</li>
+        <li>Fill out the amendment form with the required information and documents.</li>
+        <li>Complete the amendment and issue the permit.</li>
+      </ul>
+    </div>
+  );
+
   return (
+    isFeatureEnabled(Feature.ONE_WINDOW_FOR_CREATING_NEW_OR_HISTORICAL_ESUP) && parentView ? (
```

Comment:
While it doesn't hurt anything to have this included here as well as the `ExplosivesPermitFormNew`, Once it's live we'll eventually be deleting this file (and renaming the new one to this name), so it won't provide much benefit.

Reply:
Agreed.  Only thing is, with this both changes (5353 & 5431) can be controlled using two different flags.
WDYT? Removing this and controlled both features through one flag is better?

Reply:
I lean towards a single flag since it's all part of the same feature that will be released as a whole.

Reply:
Applied the changes needed for this. Please verify.